sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/instance/node/ConfigTreeNode.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.orchestration.center.instance.node;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Sets;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Config Tree Node.
+ *
+ * @author dongzonglei
+ * @author wangguangyuan
+ * @author sunbufu
+ */
+@Getter
+@Setter
+@AllArgsConstructor
+public final class ConfigTreeNode {
+    
+    private static final String SHARDING_SPHERE_KEY_ROOT = "/";
+    
+    private static final String SHARDING_SPHERE_KEY_SEPARATOR = "/";
+    
+    private static Map<String, Set<String>> childrenKeyMap = new ConcurrentHashMap<>();
```

Comment:
Is this Map only store the children of this node? If it's true, why not use Set<String> directly?

Reply:
No, The `Map` store the relationships between all the key and it's children's keys, so I use the Map structure.